src/include/daos_srv/evtree.h

```diff
@@ -198,20 +201,33 @@ struct evt_weight {
 	int64_t				wt_minor; /**< minor weight value */
 };
 
+struct evt_rect_df {
+	/** Epoch of update */
+	uint64_t	rd_epc;
+	/** Minor epoch of update */
+	uint16_t	rd_minor_epc;
+	/** High bits of length */
+	uint16_t	rd_len_hi;
+	/** Low bits of length */
+	uint32_t	rd_len_lo;
```

Comment:
can we do something like:
```
uint64_t  rd_lo;
uint64_t  rd_len:48
uint64_t  rd_minor_epc:16;
uint64_t rd_epc;

Reply:
yeah, that works.   I was a bit hesitant to use a bit field